releases/releases.yml

```diff
@@ -25,7 +25,7 @@ releases:
   uri: https://github.com/cloud-gov/uaa-customized-boshrelease
   branch: master
 - name: admin-ui
-  uri: https://github.com/cloudfoundry-community/admin-ui-boshrelease
+  uri: https://github.com/jmcarp/admin-ui-boshrelease
```

Comment:
Out of curiosity, why the switch to Josh's fork of this?

Reply:
this is reverting a change I'd made previously - `master` was even on the forks, and I didn't notice that we're actually building `deploy`

Reply:
Ah, which does have a couple of modifications not in upstream it looks like?

Reply:
correct. There should be a PR soon to nuke this release in any case